docs/docs/reference/cli.md

```diff
@@ -530,16 +532,17 @@ constellation iam create azure [flags]
 ```
   -h, --help                      help for azure
       --region string             Region the resources will be created in. (e.g. westus)
-      --resourceGroup string      Name of the resource group your IAM resources will be created in.
+      --resourceGroup string      Name prefix of the two resource groups your cluster / IAM resources will be created in.
       --servicePrincipal string   Name of the service principal that will be created.
-      --yes                       Create the IAM configuration without further confirmation
+      --yes                       Create the IAM configuration without further confirmation.
```

Comment:
Not sure what our convention is regarding the `.` at the end, but we should unify this.

Reply:
lowercase and no dot